docs/v1/P4Runtime-Spec.mdk

```diff
@@ -6284,5 +6310,21 @@ clients are aware that the server is using a larger maximum receive message
 size. The gRPC server running the P4Runtime service must not set the maximum
 receive message size to a value smaller than the default (4MB).
 
+### gRPC Default Port
+
+IANA has allocated TCP port number 9559 for the P4Runtime service, and
+P4Runtime server implementations must adopt port 9559 as the default. Users
+should be allowed to override the default using a configuration file or flag
+when starting the server. Uses of other port numbers as the default should be
+discontinued.
```

Comment:
This information is now redundant - I'm personally fine with either removing this altogether or moving the C++ sample code to "P4Runtime Service Implementation" in Section 3.

Reply:
This info is pretty obvious I think; I just added it because of the precedent in the previous sections. I will remove the whole thing.